### ISSUE TITLE
`'enable_progress_bar': False` ignored?

### ISSUE THREAD

**Issue by filippobistaffa** [unit8co/darts#991]
**Describe the bug**
`'enable_progress_bar': False` in `pl_trainer_kwargs` does not seem to produce the expected behavior.

**To Reproduce**
```
import numpy as np
import pandas as pd

from darts.dataprocessing.transformers import Scaler
from darts.datasets import AirPassengersDataset
from darts.models import TCNModel

import warnings
warnings.filterwarnings('ignore')

# Read data:
series = AirPassengersDataset().load()
series = series.astype(np.float32)

# Create training and validation sets:
train, val = series.split_after(pd.Timestamp('19590101'))

# Normalize the time series (note: we avoid fitting the transformer on the validation set)
transformer = Scaler()
train_transformed = transformer.fit_transform(train)
val_transformed = transformer.transform(val)
series_transformed = transformer.transform(series)

model = TCNModel(
    pl_trainer_kwargs={
        'enable_progress_bar': False,
    },
    n_epochs=30,
    input_chunk_length=14,
    output_chunk_length=1,
)

model.fit(
    train_transformed,
    val_series=val_transformed,
)
```

**Expected behavior**
I expected to not see the progress bar, which appears anyway.

**System (please complete the following information):**
 - Python version: 3.10.4
 - darts version: 0.19.0

**Comment by gdevos010** [comment]
`'enable_progress_bar': False` was depreciated in Pytorch lightning in deprecated is 0.7.2. 
Please use `"progress_bar_refresh_rate": 0`

**Comment by dennisbader** [comment]
Thanks, should be fixed with #999 for the coming release.

**Comment by filippobistaffa** [comment]
Thanks!
